common/scripts/ci/run_cypress_isolated_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+DIR=$(echo $(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P))
+ROOT_DIR=$(echo $(cd $(dirname "${BASH_SOURCE[0]}")/../../../ && pwd -P))
+E2E_TEST_DIR=$ROOT_DIR/libs/sdk-ui-tests-e2e
+_RUSH="${DIR}/docker_rush.sh"
+_RUSHX="${DIR}/docker_rushx.sh"
+
+# : "${SDK_BACKEND:?}"  # Todo: To be enabled later
+
+sdk_backend=$(tr <<< $SDK_BACKEND '[:upper:]' '[:lower:]')
+
+pushd $E2E_TEST_DIR
+cat > .env <<-EOF
+SDK_BACKEND=${SDK_BACKEND:-BEAR}
+TEST_WORKSPACE_ID=c76e0537d0614abb0027f7c992656b964922506f
```

Comment:
Can we use[ this way](https://github.com/gooddata/gooddata-ui-sdk/pull/3321/files#diff-2c4178351f6248c4e681d1fdfbcb1f77cc0b221e44f64138349718e89f199b2fR62) to add value of TEST_WORKSPACE_ID to `.env`